modules/sdpolicy-account-management.adoc

```diff
@@ -38,22 +40,22 @@ Customers can create, scale, and delete their clusters from link:https://cloud.r
 [id="compute_{context}"]
 == Compute
 
-Single availability zone clusters require a minimum of 4 worker nodes deployed to a single availability zone. These 4 worker nodes are included in the base subscription.
+Single availability zone clusters require a minimum of 2 worker nodes for Customer Cloud Subscription (CCS) clusters deployed to a single availability zone. A minimum of 4 worker nodes is required for standard clusters. These 4 worker nodes are included in the base subscription.
 
-Multiple availability zone clusters require a minimum of 9 worker nodes, 3 deployed to each of the three availability zones. These 9 worker nodes are included in the base subscription, and additional nodes must be purchased in multiples of three in order to maintain proper node distribution.
+Multiple availability zone clusters require a minimum of 3 worker nodes for Customer Cloud Subscription (CCS) clusters, 1 deployed to each of 3 availability zones. A minimum of 9 worker nodes are required for standard clusters. These 9 worker nodes are included in the base subscription, and additional nodes must be purchased in multiples of 3 to maintain proper node distribution.
 
-Worker nodes must all be the same type and size within a single OpenShift Dedicated cluster.
+Worker nodes must all be the same type and size within a single {product-title} cluster.
```

Comment:
```suggestion
All {product-title} clusters support a maximum of 180 worker nodes.
```
The restriction no longer applies, and this updated text reflects an update made _today_ to the Service Definition page

Reply:
Resolved

Reply:
Resolved